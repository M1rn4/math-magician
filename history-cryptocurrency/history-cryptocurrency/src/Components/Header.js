import React from 'react';
import { Link } from 'react-router-dom';
import back from '../assets/arrow_back.svg';
import '../styles/allStyles.css';

const Header = () => (
  <header>
    <div className="left">
      <Link to="/">
        <img src={back} alt="back" />
      </Link>
      <h1>CryptoChronicles</h1>
      <p>where you can view the 10 most ranked cryptocurrencies</p>
    </div>
  </header>
);

export default Header;
