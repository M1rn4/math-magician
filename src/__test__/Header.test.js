import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../functionBased/components/Header';

test('match home snapshot', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
