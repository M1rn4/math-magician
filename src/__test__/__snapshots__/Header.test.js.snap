// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match home snapshot 1`] = `
<header
  style={
    Object {
      "lineHeight": "1.5em",
      "padding": "20px 0",
    }
  }
>
  <h1
    style={
      Object {
        "color": "#000000",
        "fontSize": "6rem",
        "fontWeight": "600",
        "lineHeight": "1em",
        "marginBottom": "2rem",
        "textAlign": "center",
        "textTransform": "lowercase",
      }
    }
  >
    todos
  </h1>
</header>
`;
